secrets/secrets.go

```diff
@@ -71,30 +74,91 @@ var NewKeeper = newKeeper
 // newKeeper creates a Keeper.
 func newKeeper(k driver.Keeper) *Keeper {
 	return &Keeper{
-		k: k,
-		tracer: &oc.Tracer{
-			Package:        pkgName,
-			Provider:       oc.ProviderName(k),
-			LatencyMeasure: latencyMeasure,
-		},
+		k:      k,
+		tracer: otel.GetTracerProvider().Tracer(pkgName),
 	}
 }
 
 const pkgName = "gocloud.dev/secrets"
 
 var (
-	latencyMeasure = oc.LatencyMeasure(pkgName)
+	// Meter is the OpenTelemetry meter for this package
+	meter = otel.GetMeterProvider().Meter(pkgName)
 
-	// OpenCensusViews are predefined views for OpenCensus metrics.
-	// The views include counts and latency distributions for API method calls.
-	// See the example at https://godoc.org/go.opencensus.io/stats/view for usage.
-	OpenCensusViews = oc.Views(pkgName, latencyMeasure)
+	// latencyHistogram measures the latency of method calls
+	latencyHistogram metric.Float64Histogram
+
+	// completedCallsCounter counts the number of method calls
+	completedCallsCounter metric.Int64Counter
 )
 
+func init() {
+	var err error
+	latencyHistogram, err = meter.Float64Histogram(
+		pkgName+"/latency",
+		metric.WithDescription("Latency distribution of method calls"),
+	)
+	if err != nil {
+		otel.Handle(err)
+	}
+
+	completedCallsCounter, err = meter.Int64Counter(
```

Comment:
Why do we need this? Doesn't the latency histogram implicitly have a call count?

Reply:
@vangent I tried to maintain existing functionality here, I have no strong opinion on it and can get rid of it if its preferred not to exist
.

Reply:
In the existing functionality, I only see one metric, a "latencyMeasure"....?

Reply:
So the other "completed_calls" was being added through a view, I have tried to spend sometime to replicate the functionality.  To be able to use the latency measure as its being used now.